main.py

```diff
@@ -81,6 +83,28 @@ async def get_experiment_info(file: str) -> Any:
     return remote.examine(file)
 
 
+@app.get("/experiment/submit/")
+async def submit_experiment(file: str, args: str = "{}") -> int:
+    """Submit the given experiment file.
```

Comment:
Submits!

Reply:
Wow, all the other functions were written in singular form. I will edit these, too.